frame/executive/src/lib.rs

```diff
@@ -351,6 +355,7 @@ where
 		}
 	}
 
+	#[cfg(feature = "background-sig-check")]
```

Comment:
Please always put them below the docs, above the actual code.

Reply:
Done